contracts/mainnet/connectors/liquity/main.sol

```diff
@@ -75,25 +54,26 @@ abstract contract LiquityResolver is Events, Helpers {
             lowerHint
         );
 
-        // Allow other spells to use the borrowed amount
-        setUint(setId, borrowAmount);
-        _eventName = "LogOpen(address,uint256,uint256,uint256,uint256,uint256)";
-        _eventParam = abi.encode(msg.sender, maxFeePercentage, depositAmount, borrowAmount, getId, setId);
+        setUint(setIds[0], depositAmount);
```

Comment:
What would the use case be for setting a storage slot for the amount that was deposited?

Reply:
It might need is some heavier calculation. Let's say a user took a flashloan to deposit and now he wants to withdraw the same amount of assets from any other protocol to return to the flashloan. If the amount is sent from front end then there's no issue but the amount is calculating on the smart contract level then it matters when it's uncertain.

Also, as I mentioned above we prefer to keep things open because we don't know what other use cases could possibly be there in future.